extractor/src/test/java/org/schabi/newpipe/extractor/NewPipeTest.java

```diff
@@ -27,6 +29,21 @@ public void testAllServicesHaveDifferentId() throws Exception {
         }
     }
 
+    @Test
+    public void testGetCountryCodeAndGetLanguageCode() {
+        try {
+            HashSet<Integer> servicesId = new HashSet<>();
+            for (StreamingService streamingService : NewPipe.getServices()) {
+                Localization Localization1 = new Localization("W4JN-$nw<}7EGpwmm(EQ", "ndBdj-qEHp!#I]LDWP=,");
```

Comment:
This is unused...

Hint in general:
**Always** review your PR yourself before creating/giving it to someone for a review. Maybe also wait some minutes so your brain resets itself a bit.
Helps the later reviewers immensely and you don't embarrass yourself with trivial errors 😉